### DISCUSSION TITLE
ScriptReference/RenderTexture-width

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#362] [Notes]
# ScriptReference/RenderTexture-width

https://docs.unity3d.com/ScriptReference/RenderTexture-width.html

**Comment by HeliosDoubleSix** [comment]
If renderTexture has already been created then setting this will error: "Resizing of render texture that is loaded not supported!" you need to [`Release()`](https://docs.unity3d.com/ScriptReference/RenderTexture.Release.html) it first then you can set width/height again. This works for renderTextures that are unity asset files also.
